### ISSUE TITLE
Modifying config-HOST.php with editor messes with settings saving

### ISSUE THREAD

**Issue by demolitions** [mattpass/ICEcoder#708]
I don't think this is a bug, more like a feature request, since the saving in the "personal" config file is achieved through string searching, editing said file and changing the number of whitespaces or tabs between the array key and the assignment operator ( `=>` ) ends in a corrupted config file as soon as you open ICEcoder again, the code for saving any configuration value does not find the correct value and overwrites the initial `$ICEcoderUserSettings = array(` assignment.
Maybe that save code could be changed to not care if there's whitespace or not.
I'd label this as a feature request since there is little to no reason to go meddling manually in the config file.

**Comment by mattpass** [comment]
Agreed, a tweak that could be made in the future. Perhaps even a serialized array saved to file and unserialized when read back in, as an assoc. array.
